src/pages/guides/getting-started/installation.mdx

```diff
@@ -15,34 +15,58 @@ Please follow these links to the official installation steps for each.
 
 ## Installing the Nitric CLI
 
-### macOS
+<CodeGroup>
 
-`nitric` is available via [homebrew](https://brew.sh/)
-
-```bash
+```bash {{ title: 'macOS', label: 'nitric is available via <a href="https://brew.sh/">homebrew</a>' }}
 brew install nitrictech/tap/nitric
 ```
 
-### Windows
-
-`nitric` is available via [scoop](https://scoop-docs.vercel.app/)
-
-```
+```bash {{ title: 'Windows', label: 'nitric is available via <a href="https://scoop.sh/">scoop</a>' }}
 scoop bucket add nitric https://github.com/nitrictech/scoop-bucket.git
 scoop install nitric
 ```
 
-### Linux
+```bash {{ title: 'Linux', label: 'nitric can be downloaded as a scripted install via curl.' }}
```

Comment:
Not sure about the labels.
![image](https://github.com/nitrictech/docs/assets/13415171/46f3342b-c1f8-4ccc-ab3c-bcb4ef706d14)

The text description is a bit superfluous.

Reply:
Agree, it should be fairly obvious. I added them so it mapped one to one to the old version of doc, but happy to remove.